test/test_core.py

```diff
@@ -5599,9 +5599,8 @@ def test_files(self):
       # Sadly --closure=2 is not yet free of closure warnings
       # FIXME(https://github.com/emscripten-core/emscripten/issues/17080)
       self.ldflags.append('-Wno-error=closure')
-    elif self.maybe_closure():
-      # closure can generate variables called 'gc', which pick up js shell stuff
-      self.banned_js_engines = [config.SPIDERMONKEY_ENGINE]
```

Comment:
If we want to avoid testing closure with the spidermonkey engine we should probably do that in a more central place rather than just in this test.